### PR TITLE
Update kmesh_deploy_and_develop_in_kind.md

### PR DESCRIPTION
**Pull request by derekwin**
**What type of PR is this?**

/kind documentation

### CODE REVIEW

File: docs/kmesh_deploy_and_develop_in_kind.md [hunk 1]
```diff
@@ -121,12 +121,14 @@ You can follow the steps below to develop in kind:
     You can change the logger level by:
 
     ```shell
-    kubectl exec -it $kmesh_pod_name -n kmesh-system -- kmesh-daemon log --set default:debug
+    ./kmeshctl log $kmesh_pod_name --set default:debug
     ```
 
+   `make build` will generate kmeshctl in the kmesh directory.
```
Comment (YaoZengzeng) [inline docs/kmesh_deploy_and_develop_in_kind.md:127]: After v0.5.0 release, `kmeshctl` will be included in the release tarball, and we could use it directly without compiling it on site.

Comment (derekwin) [reply docs/kmesh_deploy_and_develop_in_kind.md]: I have removed it.

File: docs/kmesh_deploy_and_develop_in_kind.md [hunk 2]
```diff
@@ -121,12 +121,12 @@ You can follow the steps below to develop in kind:
     You can change the logger level by:
 
     ```shell
-    kubectl exec -it $kmesh_pod_name -n kmesh-system -- kmesh-daemon log --set default:debug
+    ./kmeshctl log $kmesh_pod_name --set default:debug
```
Comment (hzxuzhonghu) [inline docs/kmesh_deploy_and_develop_in_kind.md:124]:
```suggestion
    kmeshctl log $kmesh_pod_name --set default:debug
```

File: docs/kmesh_deploy_and_develop_in_kind.md [hunk 3]
```diff
@@ -121,12 +121,12 @@ You can follow the steps below to develop in kind:
...
     ```
-
+      
     Specially, for bpf logs:
     ```shell
-    kubectl exec -it $kmesh_pod_name -n kmesh-system -- kmesh-daemon log --set bpf:debug
+    ./kmeshctl log $kmesh_pod_name --set bpf:debug
```
Comment (hzxuzhonghu) [inline docs/kmesh_deploy_and_develop_in_kind.md:129]: ditto

File: docs/kmesh_deploy_and_develop_in_kind.md [hunk 4]
```diff
@@ -121,12 +121,12 @@ You can follow the steps below to develop in kind:
     You can change the logger level by:
 
     ```shell
-    kubectl exec -it $kmesh_pod_name -n kmesh-system -- kmesh-daemon log --set default:debug
+    ./kmeshctl log $kmesh_pod_name --set default:debug
     ```
-
+
```
Comment (hzxuzhonghu) [inline docs/kmesh_deploy_and_develop_in_kind.md:126]: remove

Comment (derekwin) [reply docs/kmesh_deploy_and_develop_in_kind.md]: ok

File: docs/kmesh_deploy_and_develop_in_kind.md (hunk 3 above)
Comment (derekwin) [reply docs/kmesh_deploy_and_develop_in_kind.md]: ok

**Comment by YaoZengzeng** [comment]
@derekwin you should sign-off your every commits to pass `DCO`.

**Comment by derekwin** [comment]
> @derekwin you should sign-off your every commits to pass `DCO`.

ok

**Pull request merged**